src/components/BlenderLogbook/components/BasicInfoTile.tsx

```diff
@@ -16,89 +16,76 @@ export const BasicInfoTile: React.FC<BasicInfoTileProps> = ({
   errors,
   values,
 }) => (
-  <div className="tileWrapper">
+  <div>
     <h2>Esitiedot</h2>
-    <div className="basicInfoGridRow">
-      <DropdownMenu
-        errorText={errors.divingCylinderSetId}
-        label="Pullosetti"
-        name="divingCylinderSetId"
-        type="select"
-      >
-        <optgroup label="Omat pullot">
-          {divingCylinderSets.map((dcs) => (
-            <option key={dcs.id} value={dcs.id}>
-              {/* If multiple cylinders with the same name are found, also add serial number to the name */}
-              {dcs.name}{' '}
-              {divingCylinderSets.filter((e) => e.name === dcs.name).length > 1
-                ? `(${dcs.cylinders[0].serialNumber})`
-                : ''}
+    <div className="d-flex flex-column gap-3">
+      <div className="d-flex gap-3 flex-wrap">
+        <DropdownMenu
+          disabled={values.userConfirm}
+          errorText={errors.divingCylinderSetId}
+          label="Pullosetti"
+          name="divingCylinderSetId"
+          type="select"
+        >
+          <optgroup label="Omat pullot">
+            {divingCylinderSets.map((dcs) => (
+              <option key={dcs.id} value={dcs.id}>
+                {/* If multiple cylinders with the same name are found, also add serial number to the name */}
+                {dcs.name}{' '}
+                {divingCylinderSets.filter((e) => e.name === dcs.name).length >
+                1
+                  ? `(${dcs.cylinders[0].serialNumber})`
+                  : ''}
```

Comment:
```suggestion
                {dcs.name}
                {divingCylinderSets.filter((e) => e.name === dcs.name).length >
                1
                  ? ` (${dcs.cylinders[0].serialNumber})`
                  : ''}
```
Eikai tonne haluta tollasta ylimäärästä välilyöntiä?

Reply:
Siis välilyänti olisi ylimääräinen niissä tilanteissa, että setin nimi on uniikki

Reply:
Oho joo, hyvä huomio